data/vienna-rb/vienna-rb-meetup/videos.yml

```
@@ -250,3 +250,83 @@
         00:00 Start
         00:51 Intro to logic programming
         14:48 Q&A
+
+- title: "Vienna.rb June 2025"
+  event_name: "Vienna.rb June 2025"
+  date: "2025-06-05"
+  announced_at: "2025-05-09"
+  published_at: ""
```

Might make it easier to spot when updating the event in the future 🙈 
```suggestion
  published_at: "TODO"
```